mod/display.php

```diff
@@ -93,7 +93,8 @@ function display_init(App $a)
 	}
 
 	if ($item["id"] != $item["parent"]) {
-		$item = Item::selectFirstForUser($item_user, $fields, ['id' => $item["parent"]]);
+		$parent = Item::selectFirstForUser($item_user, $fields, ['id' => $item["parent"]]);
+		$item = $parent ?: $item;
```

Comment:
When we don't have a parent (because it was deleted?) then we shouldn't display the post at all.

Reply:
Got it.